podium-gateway/src/main/webapp/app/shared/request/action-bars/request-action-toolbar/request-action-toolbar.component.ts

```diff
@@ -127,6 +130,11 @@ export class RequestActionToolbarComponent implements OnInit, OnDestroy {
         return this.requestAccessService.isRequesterOf(this.request);
     }
 
+    isReviewable(): boolean {
+        let lastFeedback = this.requestService.getLastReviewFeedbackByUser(this.request, this.currentUser);
```

Comment:
Please move access check to RequestAccessService which returns a boolean just like the other methods. 
The currentUser is already loaded in the RequestAccessService and is guaranteed to be present.